project-koku/koku
# remove filter limit/offset for tags

Changes proposed in this PR:
* Remove `limit` and `offset` from FilterSerializer for tags endpoints. These are not valid queries.

Testing Instructions:
1. Go to `http://localhost:8000/api/cost-management/v1/tags/azure/?filter[limit]=101&filter[offset]=2`
2. Expect to see this result:
```
{
    "filter": {
        "offset": [
            "Unsupported parameter or invalid value"
        ],
        "limit": [
            "Unsupported parameter or invalid value"
        ]
    }
}
```

---
Additional Context:
Previous, we would see this error response, which doesn't make sense for this endpoint:
```
{
    "error": [
        "filter[limit] and filter[offset] requires a valid group_by param."
    ]
}
```
